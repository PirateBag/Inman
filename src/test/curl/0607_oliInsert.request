
{ "rows" : [
    {  "activityState" : "INSERT", "itemId" : "1", "quantityOrdered" : "1", "quantityAssigned" : "0.0",
        "startDate" : "2025-07-01", "completeDate" : "", "orderState" : "PLANNED",
        "debitCreditIndicator" : "ADDS_TO_BALANCE" },
    {  "activityState" : "INSERT",  "itemId" : "2", "quantityOrdered" : "5", "quantityAssigned" : "0.0",
                "startDate" : "2025-0701", "completeDate" : "", "orderState" : "PLANNED",
                "debitCreditIndicator" : "ADDS_TO_BALANCE" }
] }
