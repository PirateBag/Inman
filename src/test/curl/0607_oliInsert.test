Purpose:
Add order 1 for W-001
Add Order 2 for W-002.

Both orders in the planned state, so nol components will
be assigned to orders.l

=== JSON ===
{ "rows" : [
    {  "crudAction" : "INSERT", "itemId" : "1", "quantityOrdered" : "1", "quantityAssigned" : "0.0",
        "startDate" : "2025-0701", "completeDate" : "", "orderState" : "PLANNED",
        "orderType" : "MODET" },
    {  "crudAction" : "INSERT",  "itemId" : "2", "quantityOrdered" : "1", "quantityAssigned" : "0.0",
                "startDate" : "2025-0701", "completeDate" : "", "orderState" : "PLANNED",
                "orderType" : "MODET" }
] }