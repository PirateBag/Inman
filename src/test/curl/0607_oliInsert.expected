{
  "responseType" : "MULTILINE",
  "errors" : [ ],
  "data" : [ {
    "id" : 1,
    "crudAction" : "INSERT",
    "itemId" : 1,
    "quantityOrdered" : 1.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0701",
    "completeDate" : "2025-0703",
    "parentOliId" : 0,
    "orderState" : "PLANNED",
    "orderType" : "MODET"
  }, {
    "id" : 2,
    "crudAction" : "INSERT",
    "itemId" : 2,
    "quantityOrdered" : 1.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0701",
    "completeDate" : "2025-0711",
    "parentOliId" : 0,
    "orderState" : "PLANNED",
    "orderType" : "MODET"
  } ]
}